### ISSUE TITLE
Custom buttons and demo link

### ISSUE THREAD

**Issue by Gillardo** [TerryMooreII/angular-wysiwyg#17]
I am thinking of using your wysiwyg editor which looks nice and simple.  I have used textAngular, but having some problems with it, but wondered if you editor can cope with custom buttons?  One option i would like is too open a modal window when the insert image button us pressed and allow the user to pick an image url. Can this be done?

Also your demo link on the readme doesnt work, which you may want to change..

**Comment by pieterjandesmedt** [comment]
I'm interested in custom buttons as well.
I found [bootstrap-wysiwyg editor](http://mindmup.github.io/bootstrap-wysiwyg/) to have a very elegant and lightweight solution for custom buttons. Maybe angular-wysiwyg could support such a feature?

**Comment by BharatGK** [comment]
Yes. you can add custom button like follows. but it comes with every drop as i used in drag and drop.!

+ '<div id="{{buttonId}}"  ng-if="init" class="btn btn-info btn-sm mindbowser-button"  contentEditable="{{disabled1}}"> Read More</div>'+

**Comment by agam16** [comment]
Were you able to attach the custom buttons?
